### ISSUE TITLE
TSLint rule arguments are not picked up

### ISSUE THREAD

**Issue by jeremyyap** [typescript-eslint/tslint-to-eslint-config#261]
### 🐛 Bug Report

-   `tslint-to-eslint-config` version: 0.2.10
-   ESLint version: 6.5.1
-   Node version: 12.10.0

#### Actual Behavior

The generated `.eslintrc.js` is missing some rule arguments. I believe it is due to the input TSLint rule arguments being ignored.

Actual output:
```
    "rules": {
        "@typescript-eslint/indent": "error"
    }
```

#### Expected Behavior

Expected output:
```
    "rules": {
        "@typescript-eslint/indent": [
            "error",
            2,
            {
                "FunctionDeclaration": {
                    "parameters": "first"
                },
                "FunctionExpression": {
                    "parameters": "first"
                }
            }
        ]
    }
```

#### Reproduction

Sample TSLint config:
```json
{
  "rules": {
    "align": [ true, "parameters", "statements" ],
    "indent": [ true, "spaces", 2 ]
  }
}
```

**Comment by JoshuaKGoldberg** [comment]
Ok, yes, I see it. This looks like an issue more with extended configs. Interesting how the unit tests don't cover this. Expect a new version today!

**Comment by JoshuaKGoldberg** [comment]
@jeremyyap this should have been fixed by #262, released in 0.2.11. Would you mind confirming on your end that it's fixed for you? If not, could you post your full TSLint config? Thanks!
